.github/workflows/release.yaml

```diff
@@ -0,0 +1,82 @@
+on:
+  push:
+  workflow_dispatch:
+
+env:
+  IMAGE_REGISTRY: ghcr.io
+  IMAGE_REPO: seanconroy2021/golden-container
+  IMAGE_TAGS: latest 
+
+jobs:
+  build:
+    if: ${{ github.ref == 'refs/heads/main' }}
+    runs-on: ubuntu-latest
+
+    permissions:
+      # Needed for signing the container image with GitHub OIDC Token
+      write-all
+
+    outputs:
+      image:  ${{ env.IMAGE_REGISTRY }}/${{ env.IMAGE_REPO }}
+      digest: ${{ steps.push-image.outputs.digest }}
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Install Cosign
+      uses: sigstore/cosign-installer@main
+
+    - name: Install Syft
+      uses: anchore/sbom-action/download-syft@v0.14.3
+
+    - name: Log in to ghcr
+      uses: redhat-actions/podman-login@v1
+      with:
+        registry: ${{ env.IMAGE_REGISTRY }}
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Do we need this with `redhat-actions/push-to-registry` below? I guess for cosign? Could leave a comment and nest this next to cosign below...